ovirtlago/__init__.py

```diff
@@ -133,6 +133,17 @@ def prepare_repo(
                         reposync_yum_config,
                         repos,
                     )
+        elif custom_sources:
```

Comment:
its better style to use 'is [not] Null' rather then the truthiness check. It also better UX IMO, if someone passes an empty string for the argument on the command line, he should get a "file not found" error, not the default behaviour...